src/api/resources/sync/pull.py

```diff
@@ -124,31 +156,40 @@ def get_records(viewset, revision_num=None, project=None, profile=None):
 
     :param viewset: ModelViewSet instance
     :type viewset: rest_framework.serializers.viewsets.ModelViewSet
-    :param revision_num: Revision number, defaults to None
-    :type revision_num: int, optional
-    :param project: Project id, defaults to None
-    :type project: UUID, optional
-    :param profile: Profile id, defaults to None
-    :type profile: UUID, optional
-    :return: Updates and deletes
+    :param profile_id: Profile id
```

Comment:
I think another comment or two for our future selves would help here -- I keep having to remind myself of the difference between the main `profile_id` arg and the `required_params` arg `profile`.